src/event_form/components/price-form-modal/price-form-modal-controller.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name udbApp.controller:PriceFormModalController
+ * @description
+ * # PriceFormModalController
+ * Modal for adding and editing prices.
+ */
+angular
+  .module('udb.event-form')
+  .controller('PriceFormModalController', PriceFormModalController);
+
+/* @ngInject */
+function PriceFormModalController(
+  $scope,
+  $uibModalInstance,
+  EventFormData,
+  price
+) {
+
+  var originalPrice = [];
+
+  $scope.init = init;
+  $scope.unsetPriceItemFree = unsetPriceItemFree;
```

Comment:
Gebruik view-model voor acties die nodig zijn in template: https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#controlleras-with-vm